src/main/java/com/sngular/kloadgen/randomtool/generator/ProtoBufGeneratorTool.java

```diff
@@ -47,9 +47,15 @@ public final List generateArray(final String fieldName, final String fieldType,
     return value;
   }
 
-  public final Object generateObject(final Descriptors.EnumDescriptor descriptor, final String fieldType, final List<String> fieldValuesList) {
+  public final Object generateObject(final Descriptors.EnumDescriptor descriptor, final String fieldType, final int arraySize, final List<String> fieldValuesList) {
     final List<String> parameterList = ValueUtils.replaceValuesContext(fieldValuesList);
-    return getEnumOrGenerate(descriptor, fieldType, parameterList);
+    Object value = new Object();
+    if ("enum".equalsIgnoreCase(fieldType)) {
```

Comment:
Why not creating a variable to hold those two values since you are using them several times?

Reply:
Done